tests/src/traffic/steal.rs

```diff
@@ -63,6 +71,47 @@ mod steal_tests {
         application.assert(&process).await;
     }
 
+    #[ignore] // Needs special cluster setup, so ignore by default.
+    #[rstest]
+    #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+    #[timeout(Duration::from_secs(240))]
+    async fn steal_http_ipv6_traffic(
+        #[future] ipv6_service: KubeService,
+        #[future] kube_client: Client,
+    ) {
+        let application = Application::PythonFastApiHTTPIPv6;
+        let service = ipv6_service.await;
+        let kube_client = kube_client.await;
+
+        let mut flags = vec!["--steal"];
+
+        // if cfg!(feature = "ephemeral") {
```

Comment:
We can keep it uncommented if we are planing to test the "ephemeral" variant at some point